website_sale_stock_available/readme/USAGE.rst

```diff
@@ -0,0 +1,9 @@
+To use this module, you need to:
+
+#. Go to your eCommerce store.
+#. Select a product
+#. You will see under **Add to cart** button the 'Available to promise'
+   of the product instead of 'Virtual available'.
+
```

Comment:
I have installed the module, and I see nothing under the button **Add to cart**. I need to do something else?

![image](https://user-images.githubusercontent.com/3088246/80071859-a1ae1700-8545-11ea-9e5a-53e1baa055c9.png)

Reply:
@manuelcalerosolis You set this option in the product e-commerce tab. @ernestotejeda I think you should add it to the configuration section

Reply:
I think it is essential to include that information in "USAGE.rst"